projects/plugins/jetpack/modules/account-protection.php

```diff
@@ -0,0 +1,18 @@
+<?php
+/**
+ * Module Name: Account protection
+ * Module Description: When enabled, users can only set passwords that meet strong security standards, helping protect their accounts and your site.
+ * Sort Order: 4
+ * First Introduced: $$next-version$$
```

Comment:
```suggestion
 * First Introduced: 14.4
```
This caused build errors, reverting for now but this here as a reminder that this will need to be updated.

```
[plugins/jetpack 4.266] ::error file=projects/plugins/jetpack/modules/account-protection.php,line=6::Unexpected `$$next-version$$` token.
[plugins/jetpack 5.382] Build failed: Error: Command failed with exit code 1: /home/runner/work/jetpack/jetpack/tools/replace-next-version-tag.sh -v plugins/jetpack 14.4-a.4
```